src/main/java/io/rpg/view/GameEndView.java

```diff
@@ -0,0 +1,30 @@
+package io.rpg.view;
+
+import io.rpg.viewmodel.GameEndViewModel;
+import java.io.IOException;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+
+public class GameEndView extends Scene {
+  private GameEndViewModel viewModel;
+
+  private GameEndView(Parent parent) {
+    super(parent);
+  }
+
+  public static GameEndView load() {
+    GameEndViewModel viewModel;
+    try {
+      viewModel = GameEndViewModel.load();
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
```

Comment:
I don't see the point of catching an exception, just to throw it again with a more general type, unless I'm missing something.

Reply:
We don't have to catch a RuntimeException